plugins/modules/device_credential_workflow_manager.py

```diff
@@ -1068,19 +1050,19 @@ def get_snmpV2cRead_params(self, snmpV2cReadDetails):
             data in the format suitable for the Cisco Catalyst Center config.
         """
 
-        snmpV2cRead = []
-        for item in snmpV2cReadDetails:
+        snmpV2c_read = []
+        for item in snmpV2c_read_details:
             if item is None:
-                snmpV2cRead.append(None)
+                snmpV2c_read.append(None)
             else:
                 value = {
                     "description": item.get("description"),
                     "id": item.get("id")
                 }
-                snmpV2cRead.append(value)
-        return snmpV2cRead
+                snmpV2c_read.append(value)
+        return snmpV2c_read
 
-    def get_snmpV2cWrite_params(self, snmpV2cWriteDetails):
+    def get_snmpV2cWrite_params(self, snmpV2c_write_details):
```

Comment:
Can you please check the API docstring?

get_snmp_v2c_write_params
snmp_v2c_write_details

Reply:
Modified